01-node-tutorial/answers/16-streams.js

```diff
@@ -0,0 +1,36 @@
+const { createReadStream } = require("fs");
+
+// default 64kb
+// last buffer - remainder
+// highWaterMark - control size
+// const stream = createReadStream('./content/big.txt', { highWaterMark: 200 })
+// const stream = createReadStream('../content/big.txt', { encoding: 'utf8' })
+const stream = createReadStream("../content/big.txt", {
+  highWaterMark: 200,
+  encoding: "utf8",
+});
+
+let chunkCount = 0;
+let reminder = "";
+
+//handle the "data" event
+stream.on("data", (chunk) => {
+    chunkCount++;
+    //Concate the remander from the last chunk with the current chunk
+    let data = reminder + chunk;
```

Comment:
hello, the 'reminder' and 'line' logic may not be necessary here as wasn't mentioned in the assignment.
but great job having that in mind